pkg/apis/eventsources/v1alpha1/types.go

```diff
@@ -422,6 +424,25 @@ type StripeEventSource struct {
 	EventFilter []string `json:"eventFilter,omitempty" protobuf:"bytes,5,rep,name=eventFilter"`
 }
 
+// PulsarEventSource refers to event source for Apache Pulsar related events
+type PulsarEventSource struct {
+	// Pulsar Client Configuration. For more info,
+	// see the Client and Consumer options here: http://pulsar.apache.org/docs/en/client-libraries-go/
+
+	// Service URL, e.g. "pulsar://pulsar-service:6650"
+	PulsarConfigUrl string `json:"pulsarConfigUrl" protobuf:"bytes,1,name=pulsarConfigUrl"`
```

Comment:
Can we rename these fields to simply `URL`, `ConnectionTimeout`, `Topic` etc, instead of `PulsarConfigXXXXX` as they are already part of PulsarEventSource?